### DISCUSSION TITLE
Hide winbar if not enabled?

### DISCUSSION THREAD

**Discussion by olimorris** [famiu/feline.nvim#289] [Q&A] [answered]
I currently have feline playing beautifully with `nvim-navic`. Unfortunately there are some LSPs that do not support `documentSymbols` which means I have a big hunking chunk of winbar space in Neovim. I use `set winbar=` to manually hide it but it's a bit cumbersome.

Is it possible for feline to hide winbar if it has no active elements enabled?

My truncated config is:

```lua
-- Truncated
  M.winbar_components = { active = {}, inactive = {} }

  local navic_ok, navic = om.safe_require("nvim-navic")
  if navic_ok then
    M.winbar_components.active[1] = {
      {
        provider = function()
          return navic.get_location()
        end,
        enabled = function()
          return navic.is_available()
        end,
        hl = function()
          return default_hl()
        end,
      },
    }
  end

-- Truncated

  feline.winbar.setup({
    components = M.winbar_components,
  })

```

**Comment by famiu** [comment, marked as answer]
No, and I don't plan to add support for that right now

**Reply by olimorris** [reply]
Noted. Is it a complex addition?

**Reply by famiu** [reply]
> Noted. Is it a complex addition?

Yes, at least until Neovim allows disabling winbar for specific windows
